packages/core/base/src/services/crossmintModalService.ts

```diff
@@ -139,6 +148,9 @@ export function crossmintModalService({
             if (paymentMethod) mintQueryParams.paymentMethod = paymentMethod.toLowerCase() as paymentMethods;
             if (preferredSigninMethod) mintQueryParams.preferredSigninMethod = preferredSigninMethod;
             if (prepay) mintQueryParams.prepay = "true";
+            if (layout) mintQueryParams.layout = layout ?? "popUpWindow";
```

Comment:
we don't need to pass the `layout`, the layout behavior is all within this SDK

Reply:
When we call `window.open` we should look at what layout was passed, and open the window using that layout

Reply:
@mPaella so one of the reasons we are doing this is because some clients are having the following issue:
`In their testing they found that in metamask browser the checkout popup actually takes over the current window. So once payment with crossmint is completed there is no way to get back to the original app/marketplace/minting site. `

would what you are proposing (sounds like you're saying we shouldn't have `layout` at all?) fix the issue? keep it the same?